pyatv/core/facade.py

```diff
@@ -601,6 +601,36 @@ def playstatus_error(self, updater, exception: Exception) -> None:
             self.listener.playstatus_error(updater, exception)
 
 
+class FacadeTouchGestures(Relayer, interface.TouchGestures):
+    """Facade implementation for touch gestures handling."""
+
+    def __init__(self, core_dispatcher: CoreStateDispatcher):
+        """Initialize a new FacadeTouchGestures instance."""
+        Relayer.__init__(self, interface.TouchGestures, DEFAULT_PRIORITIES)
+        interface.TouchGestures.__init__(self)
+
+    @shield.guard
+    async def touch(self, start_x: int, start_y: int, end_x: int, end_y: int, duration_ms: int) -> None:
+        """Generate a touch gesture from start to end x,y coordinates"""
+        return await self.relay("touch")(start_x=start_x, start_y=start_y,
+                                         end_x=end_x, end_y=end_y, duration_ms=duration_ms)
+
+    @shield.guard
+    async def touch_event(self, x: int, y: int, mode: HidEventMode) -> None:
```

Comment:
Maybe go with `touch_action` here? I would read "event" more like something that happened on the device rather than something that *I* did.

Reply:
Ok renamed